javascript/page_query_params.js

```diff
@@ -0,0 +1 @@
+const pageQueryParams = () => document.location.search.replace(/(^\?)/,'').split('&').reduce(function(o,n) { n=n.split('='); o[n[0]]=n[1]; return o},{});
```

Comment:
The reduce function
> function(o,n) { n=n.split('='); o[n[0]]=n[1]; return o}

Is actually three lines 🤔

```js
  function(o,n) {
    n=n.split('=');
    o[n[0]]=n[1];
    return o
  }
```
as per defined in #19 (>(i.e. no using ; instead of a new line))
To make this one line though I wonder if this is possible with chained maps/reduces 🤔